README.md

```diff
@@ -477,17 +477,10 @@ Dock][icon-mac-instructions].
 
 # Update
 
-Spacemacs supports two different update schemes, the default is a rolling update
-scheme based on the latest version of packages available. This version can be
-found on the `develop` branch and is updated by a simple `git pull`.
+Spacemacs relies solely on a rolling update scheme based on the latest 
+version of packages available. To update Spacemacs, simply pull the latest
+changes of the default `develop` branch: 
```

Comment:
"default" causes confusion. because previously both develop and master  branches are mentioned, but now only the develop branch is. So it's not clear to user why "default" is emphasized. Is there other non-default branches?

Reply:
lets remove default and just mention develop.